### DISCUSSION TITLE
essay/2024/06/13/ai-developer

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#216] [Comments]
# essay/2024/06/13/ai-developer

AI가 개발을 점점 잘하게 되면서 모든 개발자들이 걱정입니다. 앞으로 세상은 어떻게 되려나. 다들 걱정하는 가운데 오히려 희망을 품고 있는 사람들도 있습니다.

https://jeho.page/essay/2024/06/13/ai-developer.html

**Comment by bookseal** [comment]
차이는 오랜 시간의 땀과 노력이죠.

**Reply by BenjaminKim** [reply]
정말 그렇습니다..!
